src/concrete/ml/onnx/ops_impl.py

```diff
@@ -893,11 +897,17 @@ def numpy_equal(
     Args:
         x (numpy.ndarray): Input tensor
         y (numpy.ndarray): Input tensor
+        lsbs_to_remove (Optional[int]): The number of the least significant bits to remove.
```

Comment:
I would remove the second `the` in `The number of the least significant bits to remove.`